src/isaw.theme/isaw/theme/static/theme.css

```diff
@@ -717,6 +717,23 @@ body #content ul.news.simple a.TileLink:hover, body #content ul.news.simple a.Ti
     margin-top: 8px;
 }
 
+/* Folder summary */
+body.template-folder_summary_view #content h2.tileHeadline:has(+ div + p + p) {
```

Comment:
This selector doesn't work in Firefox, IIRC. Maybe we can handle this another way? It seems like putting no bottom margin on the internal paragraph(s) and 28px top margin all on the h2, might suffice.